code/game/objects/structures/morgue.dm

```diff
@@ -204,6 +204,7 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
 			if(mob_occupant.stat == DEAD && beeper && COOLDOWN_FINISHED(src, next_beep))
 				playsound(src, 'sound/weapons/gun/general/empty_alarm.ogg', 50, FALSE) //Revive them you blind fucks
 				COOLDOWN_START(src, next_beep, beep_cooldown)
+				SEND_SIGNAL(src, COMSIG_MORGUE_ALARM, mob_occupant) // SKYRAT EDIT ADDITION: MORGUE RADIO
```

Comment:
This needs to be on its own cooldown, spamming it every five seconds will be too much. It should be at most like, once every minute, or something like that.

Reply:
on cooldown now